developer/src/common/web/utils/src/xml-utils.ts

```diff
@@ -184,7 +229,7 @@ export class KeymanXMLReader {
           }
         }
       });
-      return Object.fromEntries(e);
+      return this.copySymbols(Object.fromEntries(e), data);
```

Comment:
copySymbols/removeSymbols aren't xml specific, would they be better elsewhere (probably still in common/web/utils)?